modules/understanding-vsphere-problem-detector.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/using-vsphere-problem-detector-operator.adoc
+
+:operator-name: vSphere Problem Detector Operator
```

Comment:
I could swear I heard or read not to use a substitution in a heading.  I used an attribute because A) I wasn't sure of the final name when I started, and B) I grew tired of the idea of rekeying it each time.

Reply:
Fair enough. And no an attribute in a heading shouldn't be a problem. Just searched the repo (`grep -r "^=.*{" | wc -l`) and see 280 hits across existing modules that use 'em.

Reply:
Then I'll pop it in the heading.  I prefer consistency but thought I was cornered into _not_ using the attr.